### ISSUE TITLE
Cannot install rustface even when using nightly Rust

### ISSUE THREAD

**Issue by rococtz** [atomashpolskiy/rustface#7]
Hi,

I am using the nightly build of Rust:
```
rustup 1.11.0 (e751ff9f8 2018-02-13)
rustc 1.28.0-nightly (f28c7aef7 2018-06-19)
cargo 1.28.0-nightly (e2348c2db 2018-06-07)
```

but when I try to install rustface, I get this error:
```
232 | #[target_feature = "+tbm"]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^

error: #[target_feature] attribute must be of the form #[target_feature(..)]
   --> /Users/user/.cargo/registry/src/github.com-1ecc6299db9ec823/stdsimd-0.0.3/src/x86/tbm.rs:244:1
    |
244 | #[target_feature = "+tbm"]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^

error: #[target_feature] attribute must be of the form #[target_feature(..)]
   --> /Users/user/.cargo/registry/src/github.com-1ecc6299db9ec823/stdsimd-0.0.3/src/x86/tbm.rs:255:1
    |
255 | #[target_feature = "+tbm"]
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 750 previous errors

error: Could not compile `stdsimd`.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `rustface v0.1.0`, intermediate artifacts can be found at `/var/folders/pz/45htc8zn6_sbd3411btr4pq40000gp/T/cargo-installGrNVuk
```

**Comment by atomashpolskiy** [comment]
Seems to me like current simd version is outdated and incompatible with current Rust version

**Comment by rococtz** [comment]
The guys from `stdsimd` say rustface should not use the `stdsimd` from crates.io, but from master (issue raised [here](https://github.com/rust-lang-nursery/stdsimd/issues/488))

**Comment by atomashpolskiy** [comment]
Would you be able to make a PR? Thanks for investigating!

**Comment by gnzlbg** [comment]
@atomashpolskiy `core::arch` `x86` and `x86_64` components are available on stable Rust 1.27, so if you upgrade the minimum Rust version of `rustface` to that you can drop the dependency on `stdsimd` and just use it via the `std` library. 

All other `core::arch` components are available on nightly via the std library as well. The `core::simd` component is available on nightly via the std library too (`core::simd`). 

Iff you need to use `stdsimd`'s master branch for some reason, I've updated the `stdsimd` readme with instructions on how to do so, but that is extremely unstable and will break often.

The `crates.io` versions of `stdsimd` are deprecated.

**Comment by jjhbw** [comment]
The `stdsimd` dependency is now removed from rustface master, so it should now compile with stable rustc. See #9.

If you want to try the version with explicit SIMD and compile on nightly, instructions for that are in the readme.
